XXX: use fmt::format while clang doesn't support std::format yet

Index: rts/System/Sync/DumpHistory.cpp
--- rts/System/Sync/DumpHistory.cpp.orig
+++ rts/System/Sync/DumpHistory.cpp
@@ -1,6 +1,7 @@
 /* This file is part of the Spring engine (GPL v2 or later), see LICENSE.html */
 
 #include <fstream>
+#include <fmt/core.h>
 
 #include "DumpHistory.h"
 #include "SyncChecker.h"
@@ -36,13 +37,13 @@ void DumpHistory(std::fstream& file, int frameNum, boo
 
 	LOG("[%s] dump history at %d (%d records)", __func__, frameNum, firstRangeSize + secondRangeSize);
 
-	file << std::format("internal frame checksums ({}):\n", frameNum);
+	file << fmt::format("internal frame checksums ({}):\n", frameNum);
 
 	for (int i = startIndex; i < startIndex + firstRangeSize; ++i) {
-		file << std::format("\t{:08x}\n", data[i]);
+		file << fmt::format("\t{:08x}\n", data[i]);
 	}
 	for (int i = 0; i < secondRangeSize; ++i) {
-		file << std::format("\t{:08x}\n", data[i]);
+		file << fmt::format("\t{:08x}\n", data[i]);
 	}
 #endif // SYNC_HISTORY
 }
